packages/devops/scripts/lambda/createOrRestoreDeployment.py

```diff
@@ -228,37 +210,32 @@ def lambda_handler(event, context):
 
     filters = [
         { 'Name': 'tag-key', 'Values': ['RestoreFrom'] },
-        { 'Name': 'instance-state-name', 'Values': ['running'] }
+        { 'Name': 'instance-state-name', 'Values': ['stopped'] }
```

Comment:
Does this mean you can only restore from stopped instances?

Reply:
That's right - if you want to restore an existing instance, you have to stop it first. I don't think people ever really do this in practice, so just kept the code clean and easy by restricting it to our expected overnight case - they'll be stopped.